### ISSUE TITLE
Builds without "--enable-gpl" and "--enable-nonfree"

### ISSUE THREAD

**Issue by alexhx5** [vot/ffbinaries-prebuilt#6]
Would it be possible for you to also provide builds without `--enable-gpl` and `--enable-nonfree`?

The way zeranoe does it. For example:
`ffmpeg-4.2.1-win64-shared-lgpl.zip` at https://ffmpeg.zeranoe.com/builds/win64/shared/

It is advised on the [ffmpeg page](https://www.ffmpeg.org/legal.html) to use such builds for redistributing ffmpeg in order to comply with the licenses.

**Comment by vot** [comment]
Hi @AlexSHoffman,

I would absolutely love to do that - in fact it's one of the long-standing to-do items of this project.

The way I'd like this to be done is to get the Docker builds going for the ffmpeg suite (which also allows for nightly builds to be available). This way we'd get consistent sets of binaries, across platforms, with full choice of which licensing flavour you'd like :)

Existing issue about nightly builds:
#2 

TL;DR version: It will need to be done but I never had time (and I'm hoping for someone to jump in and help me with setting this up). It's the top of my wishlist but unfortunately I can't provide a target date yet.
